File: 프로그래머스/lv1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
import java.util.ArrayList;
import java.util.Arrays;

class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        ArrayList<Integer> space = new ArrayList<>(Arrays.asList(1,2,3,4,5,6,7,8,9));
        for (int i =0; i<numbers.length; i++){
            space.remove(Integer.valueOf(numbers[i]));
        }
        for (int i = 0; i< space.size(); i++){
            answer += space.get(i);
        }
        return answer;
    }
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.py
def solution(numbers):
    answer = 0
    dic = {}
    
    for i in range(10):
        dic[i] = 0
        
    for i in numbers:
        dic[i] += 1
        
    for k,v in dic.items():
        if v == 0:
            answer += k
    return answer